bin/kona-exex/src/main.rs

```diff
@@ -1,7 +1,29 @@
-use eyre::Result;
+use std::sync::Arc;
 
-#[tokio::main]
-async fn main() -> Result<()> {
-    println!("Hello, kona!");
-    Ok(())
+use clap::Parser;
+use eyre::{bail, Result};
+use reth::cli::Cli;
+use reth_node_ethereum::EthereumNode;
+use superchain_registry::ROLLUP_CONFIGS;
+
+mod kona;
+use kona::KonaExEx;
+
+mod cli;
+use cli::KonaArgsExt;
+
+/// The identifier of the Kona Execution Extension.
+const EXEX_ID: &str = "kona";
+
+fn main() -> Result<()> {
+    Cli::<KonaArgsExt>::parse().run(|builder, args| async move {
+        let Some(cfg) = ROLLUP_CONFIGS.get(&args.l2_chain_id).cloned() else {
```

Comment:
in the future would it be worth allowing rollup configs outside of the registry using a cli param or file (maybe for ephemeral devnets or something)? if so would make an issue for it

Reply:
I think so, for stuff like kurtosis specifying a path to a json file should be a good idea.
This should live inside the superchain-primitives crate imo, if so i'll make open an issue, cc @refcell